pyproject.toml

```diff
@@ -146,3 +139,12 @@ ignore = [
     "D406", # Section name should end with a newline
     "D407", # Missing dashed underline after section
 ]
+
+[tool.rstcheck]
+ignore_directives = [
+    "automodule",
+    "mdinclude",
```

Comment:
much better than what I did in previously in the `.pre-commit-config.yml`, thanks!